File: monitor/alerts/list-get-example-all/list-get-example-all.rb
# Get twilio-ruby from twilio.com/docs/ruby/install
require 'rubygems'          # This line not needed for ruby > 1.8
require 'twilio-ruby'

# Get your Account Sid and Auth Token from twilio.com/user/account
account_sid = 'ACda6f132a3c49700934481addd5ce1659'
auth_token = '{{ auth_token }}'
@client = Twilio::REST::MonitorClient.new account_sid, auth_token


# Loop over alerts and print out a property for each one
@client.account.alerts.list.each do |alert|
    puts alert.alert_text
end

File: monitor/alerts/list-get-example-all/list-get-example-all.php
<?php
// Get the PHP helper library from twilio.com/docs/php/install
require_once('/path/to/twilio-php/Services/Twilio.php'); // Loads the library

// Your Account Sid and Auth Token from twilio.com/user/account
$sid = "ACda6f132a3c49700934481addd5ce1659";
$token = "{{ auth_token }}";
$client = new Monitor_Services_Twilio($sid, $token);

// Loop over the list of alerts and echo a property for each one
foreach ($client->alerts as $alert) {
    echo $alert->alert_text;
}